flink-state-backends/flink-statebackend-rocksdb/src/main/java/org/apache/flink/state/rocksdb/RocksDBStateUploader.java

```diff
@@ -180,6 +190,19 @@ private HandleAndLocalPath uploadLocalFileToCheckpointFs(
         }
     }
 
+    @VisibleForTesting
+    void applyJitter(Supplier<Long> sleepTime) throws InterruptedException {
```

Comment:
You are never actually using this parameter to pass in something else. I think passing in `Consumer<Long>` and using `sleepTime -> Thread.sleep(sleepTime)` by default would make more sense. You can then pass in a different Consumer in the tests, e.g. `sleepTime -> assertThat(sleepTime).isEqualTo(..)`.

Reply:
Good idea. It is the best way to assert the internal logic without define a return value.